Sources/DDGSync/DDGSync.swift

```diff
@@ -171,9 +166,23 @@ public class DDGSync: DDGSyncing {
         }
     }
 
+    public var serverEnvironment: ServerEnvironment {
+        if dependencies.endpoints.baseURL == ServerEnvironment.production.baseURL {
+            return .production
+        }
+        return .development
+    }
+
+    public func updateServerEnvironment(_ serverEnvironment: ServerEnvironment) {
+        try? updateAccount(nil)
+        dependencies.updateServerEnvironment(serverEnvironment)
+        authState = .initializing
+        initializeIfNeeded(isInternalUser: false)
```

Comment:
I think it could be a good time to get rid of `isInternalUser` parameter and that old migration, wdyt?

Reply:
Right, good idea. Let me do this.